backend/src/main.rs

```diff
@@ -11,6 +11,7 @@ use std::{net::SocketAddr, time::Duration};
 use tower_http::{add_extension::AddExtensionLayer, cors::CorsLayer, trace::TraceLayer};
 use tracing_subscriber::EnvFilter;
 
+mod queries;
```

Comment:
```suggestion
mod db;
```
And change the directory to be `db` instead of `queries`.  And move the `pub struct Db;` definition to the `db/mod.rs`